stl/inc/format

```diff
@@ -2208,6 +2208,9 @@ private:
         _Out&& _OutputIt_, const basic_format_args<basic_format_context>& _Ctx_args, const _Lazy_locale& _Loc_)
         : _OutputIt(_STD move(_OutputIt_)), _Args(_Ctx_args), _Loc(_Loc_) {}
 
+    basic_format_context(const basic_format_context&)            = delete;
+    basic_format_context& operator=(const basic_format_context&) = delete;
+
 public:

```

Comment:
This correctly implements the resolution of LWG-4061 but I think the resolution has a minor issue - deleted copies should be public, not private. That way, if the user tries to copy something, they get an error about a deleted ctor/assign, instead of an access control error.

(This is a difference from the old technique of "private and unimplemented".)

Reply:
I think it doesn't matter which access control deleted functions have. When a private and deleted overload is selected, currently no complier emits error message for being private ([Godbolt link](https://godbolt.org/z/xqWPTqv8n)).

Reply:
Interesting, I could have sworn it was different years ago. Thanks!